kcidev/subcommands/results.py

```diff
@@ -208,8 +271,14 @@ def results(ctx, origin, giturl, branch, commit, action, download_logs, latest,
         cmd_list_trees(origin)
     elif action == "builds":
         if not giturl or not branch or not ((commit != None) ^ latest):
-            kci_err("--giturl AND --branch AND (--commit XOR --latest) are required")
-            raise click.Abort()
+            kci_msg("Getting git repository informations for the current folder")
+            git_url, branch_name, last_commit_hash = get_folder_repository(git_folder)
+            kci_msg(git_url)
+            kci_msg(branch_name)
+            kci_msg(last_commit_hash)
```

Comment:
I'd improve printing here:
```
tree: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git
branch: master
commit: 2144da25584eb10b84252230319b5783f6a83041
```

and maybe move the `kci_msg()` calls, so all you are left with is:

` + giturl, branch, commit = get_folder_repository(git_folder)`

Reply:
nice catch!

Reply:
fixed in last commit